packages/components/src/filters/advanced/README.md

```diff
@@ -84,7 +84,7 @@ input: {
 ```
 
 `type`: A string Autocompleter type used by the [Search Component](https://github.com/woocommerce/wc-admin/tree/master/packages/components/src/search).
-`getLabels`: A function returning a Promise resolving to an array of objects with `id` and `label` properties.
+`getLabels`: Either an array of objects with `id` and `label` properties, or a function returning a Promise resolving to an array of objects with `id` and `label` properties.
```

Comment:
This can be reverted now. I didn't know you could return an `async` function as you did 👍 . I thought it would need to be `return Promise.resolve( [] );`, so maybe that can be described here?